### PR TITLE
Updating SDK to 1.3.1

### PR DESCRIPTION
**Pull request by Journie-M**
Also updates node LTS.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (Integralist) [inline .github/CODEOWNERS]: Hmm. So GitHub is complaining that this team either doesn't exist or should have write access to the repo.

I found the team, so they do exist, and I've added them to this repo with write access BUT there are _zero_ members of that team so no one is going to get notified 😬

**Review by Integralist — APPROVED** [review]
Approved to unblock but there is a possible issue with the CODEOWNER file (see comment).

File: .github/CODEOWNERS
Comment (doramatadora) [reply .github/CODEOWNERS]: Ok, this team is a codeowner on other starter kits, so I will raise it in the appropriate slack channel.

Comment (doramatadora) [reply .github/CODEOWNERS]: Removing for now

**Pull request merged**
